### ISSUE TITLE
[Enahancment] link songs with steam install of the game 

### ISSUE THREAD

**Issue by hax4dazy** [RiskiVR/BSLegacyLauncher#61]
Instead of making a folder in the root of the Beat Saber Legacy Launcher, make it so that the option in advanced settings links the folders with the current install of Beat Saber.

**Comment by RiskiVR** [comment]
There is no way of detecting where your steam install of the game is, as it's installed differently for everyone. 
I may add an external manual option for this.

**Comment by UrkiMimi** [comment]
I mean you could also symlink the CustomLevels folder manually like I did for pretty much every Beat Saber instance that isn't my main one. [This is the guide I followed.](https://www.howtogeek.com/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/)

**Comment by hax4dazy** [comment]
Yeah I'm aware that you are able to do that but it would be more convenient if you were able to specify where the main install of Beat Saber is and it links the folders to all the installs instantly.
